airbyte-config/config-models/src/main/java/io/airbyte/config/EnvConfigs.java

```diff
@@ -312,12 +310,12 @@ public AirbyteVersion getAirbyteVersion() {
 
   @Override
   public Version getAirbyteProtocolVersionMax() {
-    return new Version(getEnvOrDefault(AIRBYTE_PROTOCOL_VERSION_MAX, "0.3.0"));
+    return new Version("0.3.0");
```

Comment:
Micronaut is expecting an envvar in the server it default to those values. But I think that we should check is other micronaut application.yml also specify a default value and don't rely on the return of this value.

Reply:
https://github.com/airbytehq/airbyte/blob/master/airbyte-server/src/main/resources/application.yml#L90

Reply:
When we get rid of `EnvConfigs` we should directly use the values from the micronaut config. Not sure where we are on this, this is a quick fix to remove logging noise.